scripts/testnet/include/nodes.sh

```diff
@@ -120,19 +124,23 @@ iterateOverValidators() {
   local VALIDATOR_INDEX=0
 
   # Iterate over Shard Validators
-  let "max_shard_id=$SHARDCOUNT - 1"
-  for SHARD in `seq 0 1 $max_shard_id`; do
-    for VALIDATOR_IN_SHARD in `seq $SHARD_VALIDATORCOUNT`; do
-      $callback $SHARD $VALIDATOR_INDEX
-      let VALIDATOR_INDEX++
+  (( max_shard_id=$SHARDCOUNT - 1 ))
+  for SHARD in $(seq 0 1 $max_shard_id); do
+    for _ in $(seq $SHARD_VALIDATORCOUNT); do
+      if [ $VALIDATOR_INDEX -ne $SKIP_VALIDATOR_IDX ]; then
+        $callback $SHARD $VALIDATOR_INDEX
+      fi
+      (( VALIDATOR_INDEX++ ))
```

Comment:
Why the replacement of `let` with `(( ))`? Not a problem, just asking. Replacement of back-ticks with `$()` is a good idea though.

Reply:
Goland bash linter suggestion